utils/zerovec/src/zerovec/mod.rs

```diff
@@ -353,7 +353,7 @@ where
     pub const unsafe fn from_bytes_unchecked(bytes: &'a [u8]) -> Self {
         // &[u8] and &[T::ULE] are the same slice with different length metadata.
         /// core::slice::from_raw_parts(a, b) = core::mem::transmute((a, b)) hack
-        /// ```compile_fail
+        /// ```
```

Comment:
ditto

Reply:
Same as in `slice.rs`, although that's the same code so maybe should be deduped.